tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/contact_call_to_action.html

```diff
@@ -1,18 +1,22 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 <div class="contact-cta grid__contact-cta">
-    {% for cta in value.call_to_action %}
-        <h2 class="contact-cta__text heading heading--three heading--light">{{ cta.value.text }}</h2>
-        {% if cta.value.description %}{{ cta.value.description|richtext }}{% endif %}
-    {% endfor %}
-
     {% if value.person %}
-        {% include "patterns/atoms/avatar/avatar.html" with avatar=value.person.image classes="contact-cta__avatar avatar--small" %}
+        {% include "patterns/atoms/avatar/avatar.html" with avatar=value.person.image classes="contact-cta__avatar avatar--cta" %}
     {% endif %}
 
+    <div class="contact-cta__text">
+        {% for cta in value.call_to_action %}
+            <h2 class="heading heading--two-b">{{ cta.value.text }}</h2>
+            {% if cta.value.description %}
+                <div class="contact-cta__description">{{ cta.value.description|richtext }}</div>
+            {% endif %}
+        {% endfor %}
+    </div>
+
     {% if value.person %}
         <p class="contact-cta__cite">
             {% if value.person.name %}
-                <span class="contact-cta__name">{{ value.person.name }}</span>
+                <span class="contact-cta__name mr-2">{{ value.person.name }}</span>
```

Comment:
Suggestion: I wonder if it's worth adding this to `<p class="contact-cta__cite">` to give it a bit of space so it doesn't get too close to the button:

<img width="719" alt="image" src="https://github.com/user-attachments/assets/4834df42-d405-46c3-b104-728a09777c63" />